.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: icu-date-scm-1.rockspec
+
+  publish-tag:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Create a rockspec for the release.
+      - run: printf '%s=%s\n' TAG "${GITHUB_REF##*/}" >> "${GITHUB_ENV}"
+      - run: sed -E
+          -e 's/branch = ".+"/tag = "${{ env.TAG }}"/g'
```

Comment:
The single quotes are used here. Please, replace `-e 's/branch = ".+"/tag = "${{ env.TAG }}"/g'` with `-e "s/branch = '.+'/tag = '${{ env.TAG }}'/g"`. Keep the second sed expression unchanged: double quotes are used here.

Reply:
Fixed. Thanks.